data/data-pipeline/data_pipeline/etl/sources/cdc_svi_index/etl.py

```diff
@@ -0,0 +1,138 @@
+            "E_AGE65",  # Persons aged 65 and older estimate, 2014-2018 ACS, S0101_C01_030 E
+            "E_DISABL",  # Civilian noninstitutionalized population with a disability estimate, 2014 -2018 ACS, DP02_0071E
+            "E_SNGPNT",  # Single parent household with children under 18 estimate MOE, 2014 -2018 ACS, (DP02_0007E + DP02_0009E)
+            "E_CROWD",  # At household level (occupied housing units), more people than rooms estimate for 2014 -2018 ACS, (DP04_0078E + DP04_0079E)
+        ]
+
+        self.COLUMNS_TO_KEEP = [
+            self.GEOID_TRACT_FIELD_NAME,
+            field_names.CDC_SVI_INDEX_SE_THEME_PERCENTILE_FIELD,  # SE prefix references "socioeconomic"
+            field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_PERCENTILE_COMPOSITION_FIELD,
+            field_names.CDC_SVI_INDEX_LANGUAGE_THEME_PERCENTILE_FIELD,
+            field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_PERCENTILE_FIELD,
+            field_names.CDC_RPL_THEMES_OVERALL_PERCENTILE_FIELD,
+            # Sum of series fields
+            field_names.CDC_SVI_INDEX_SE_THEME_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_LANGUAGE_THEME_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_SOS_FIELD,
+            field_names.CDC_SVI_INDEX_THEMES_OVERALL_SOS_FIELD,
+            # Derived columns
+            field_names.CDC_SVI_INDEX_THEMES_OVERALL_TOTAL_FIELD,
+            field_names.CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY,
+        ] + self.COVARIATE_COLUMNS
+
+        self.df: pd.DataFrame
+
+    def extract(self) -> None:
+        logger.info("Extracting 43 MB CDC SVI INDEX")
+        super().extract(
+            self.CDC_SVI_INDEX_S3_URL,
+            self.CDC_SVI_INDEX_ZIP_FILE_DIR,
+        )
+
+    def transform(self) -> None:
+        logger.info("Starting CDC SVI INDEX transform")
+
+        # New file name
+        tmp_svi_csv_file_path = (
+            self.CDC_SVI_INDEX_ZIP_FILE_DIR / "SVI2018_US.csv"
+        )
+
+        self.df = pd.read_csv(
+            filepath_or_buffer=tmp_svi_csv_file_path,
+            dtype={self.CDC_SVI_INDEX_TRACTS_FIPS_CODE: object},
+        )
+
+        # Note: In this dataset all US census tracts are ranked against one another.
+        # Puerto Rico is not included in this dataset
+        self.df.rename(
+            columns={
+                self.CDC_SVI_INDEX_TRACTS_FIPS_CODE: self.GEOID_TRACT_FIELD_NAME,
+                "RPL_THEME1": field_names.CDC_SVI_INDEX_SE_THEME_PERCENTILE_FIELD,
+                "RPL_THEME2": field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_PERCENTILE_COMPOSITION_FIELD,
+                "RPL_THEME3": field_names.CDC_SVI_INDEX_LANGUAGE_THEME_PERCENTILE_FIELD,
+                "RPL_THEME4": field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_PERCENTILE_FIELD,
+                "RPL_THEMES": field_names.CDC_RPL_THEMES_OVERALL_PERCENTILE_FIELD,
```

Comment:
In light of concerns regarding the measures in CDC Place, this `RPL_THEMES` is a function of "Minority and Language Themes" - comprised of ACS 2018 variables that use variables that references race. `RPL_THEMES` is subsequently used to flag priority communities. I would like to discuss this as this may not be palatable with our stakeholders.